tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+import pytest
+from app.database.database import Base, SessionLocal, engine
+from app.database.models import Event, User
+from app.main import app
+from fastapi.testclient import TestClient
+
+
+@pytest.fixture
+def client():
+    return TestClient(app)
+
+
+@pytest.fixture
+def sessions():
```

Comment:
Looks very specific to your tests :)
Maybe move it to less general file?

Reply:
The idea of "session" fixture is good.

Let's separate it from "user" fixtures (maybe use hypothesis to create random users.)

Reply:
Tried with the hypothesis module, found out that it doesn't go well with pytest.fixture.
Instead, I'm going to use Faker library for email and username.